Configurations.md

```diff
@@ -2428,6 +2428,68 @@ Break comments to fit on the line
 // commodo consequat.
 ```
 
+## `wrap_long_multiline_chains`
+
+Wrap the right hand side of a let binding when it is a long multiline chain.
+Leave other right hand expressions alone.
+
+- **Default value**: false
+- **Possible values**: true, false
+- **Stable**: Yes
```

Comment:
Except in some rarer circumstances, net-new configuration options should be added as unstable.